src/utilities/makeDraftModule.py

```diff
@@ -321,7 +323,21 @@ def createCppModule(self):
             headerFile += '    Message<' + msg['type'] + 'Payload> ' + msg['var'] \
                           + ';  //!< ' + msg['desc'] + '\n'
         headerFile += '\n'
-        headerFile += '    BSKLogger bskLogger;              //!< -- BSK Logging\n'
+        headerFile += '    BSKLogger bskLogger;              //!< BSK Logging\n'
+        headerFile += '\n'
+        if len(variableList):
+            for msg in variableList:
+                varName = msg['var'];
+                headerFile += ("    /** setter for `" + varName + "` property */\n")
```

Comment:
I'm generally of the opinion that fstrings are easier to read and maintain than string addition, especially in cases where ensuring that spaces are correct is critical. For example:
```
headerFile += (" /** setter for `" + varName + "` property */\n")
```
could be 
```
headerFile += f" /** setter for `{varName}` property */\n"
```
and line 335 
```
headerFile += ('    ' + msg['type'] + ' get' + varName[:1].upper() + varName[1:] \
              + '() const {return this->' + varName + ';}\n')
```
could be
```
headerFile += f'    {msg['type']} get{varName[:1].upper()}{varName[1:]}() const {{return this->'{varName}';}}\n'
```
The latter shows the one downside of the fstring approach, which is that `{` and `}` get escaped as `{{` and `}}` which may be misleading.

Reply:
Also, `{varName[:1].upper()}{varName[1:]}` is equivalent to `{varName.capitalize()}`

Reply:
I did try `capitalize()`.  However, it just capitalizes the first letter and makes the rest of the variable name lower case.  I'm looking to preserve the lower camel case spelling, but just need to ensure the first letter is a capital letter.

Reply:
Howdy @Mark2000 , thanks for the feedback.  I'm not seeing the benefit of adding strings one way or another.  Can you elaborate?  The formatting is correct and human readable in either case, plus we don't need the `{{` etc?  Thanks for your thoughts on why you feel strings are better?

Reply:
Re: capitalize, didn't realize that was the case. Your way makes sense then.

Reply:
I agree that the strings as-is are correct, but I think that for ``"some text `" + quotedVar + "` some text"``it is less obvious that `quotedVar` has backticks around it than ``f"some text `{quotedVar}` some text"``. I also think that it is less obvious which strings and variables have spaces around them vs. concatenations of words in `"concat1" + concat2 + " separate"` as opposed to `f"concat1{concat2} separate"`. In my opinion, those benefits outweigh the drawback of using `{{` for `{`.

A rather extreme example would be producing the string ``A) `(1 + 2) = 3` is addition.`` given some variables `a=1`, `b=2`, `method="addition"`. The output of the latter is obvious from the code, while the former is unclear to the code reader. 
```
my_string = "A) `(" + str(a) + " + " + str(b) + ") = " + str(a + b) + "` is " + method + ".\n"
```
as opposed to
```
my_string = f"A) `({a} + {b}) = {a+b}` is {method}.\n"
```

Reply:
That does really help with seeing the indentations; wasn't familiar with `string.Template` before. I think the only downside is having the template defined far away from where it's actually used is a little unintuitive, but makes sense to have it outdented all the way (and of course if everything was in that style and the generation functions were a bit more atomic it wouldn't be an issue).

Reply:
Howdy Mark, thanks for letting me know about `strings`.  I'll work on replacing text strings assembled with `+` with fstrings in this script.

Reply:
Ok, I restored the original, direct, way of defining `std::array` swig interfaces.  Much easier to read and maintain ;-) 

@Mark2000 , I updated the `makeDraftModule.py` methods to use `fstrings` instead of regular strings.  I did a folder comparison of before and after of the auto-generated draft modules and they remain identical.  All tests passed.